SPECS/harfbuzz/harfbuzz.spec

```diff
@@ -1,45 +1,53 @@
-Summary:	opentype text shaping engine
-Name:		harfbuzz
-Version:	1.9.0
-Release:    4%{?dist}
-License:	MIT
-URL:		http://harfbuzz.org
-Group:		System Environment/Libraries
+Summary:        opentype text shaping engine
+Name:           harfbuzz
+Version:        2.6.4
+Release:        4%{?dist}
+License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:	    https://www.freedesktop.org/software/harfbuzz/release/%{name}-%{version}.tar.bz2
-BuildRequires:	glib-devel
-BuildRequires:	freetype
-BuildRequires:	freetype-devel
-Requires:	    glib
+Group:          System Environment/Libraries
+URL:            https://harfbuzz.github.io/
+Source0:        https://www.freedesktop.org/software/harfbuzz/release/%{name}-%{version}.tar.xz
+BuildRequires:  pkg-config
+BuildRequires:  pkgconfig(fontconfig)
+BuildRequires:  pkgconfig(freetype2)
+BuildRequires:  glib-devel
+BuildRequires:  pkgconfig(icu-uc)
+%if %{with_check}
+BuildRequires:  python3-devel
+BuildRequires:  which
+BuildRequires:  binutils
+%endif
+Requires:       glib
 
 %description
 HarfBuzz is an implementation of the OpenType Layout engine.
 
-%package	devel
-Summary:	Header and development files
-Requires:	%{name} = %{version}-%{release}
-Requires:	glib-devel
+%package        devel
+Summary:        Header and development files
+Requires:       %{name} = %{version}-%{release}
+Requires:       pkgconfig(glib-2.0)
+Provides:       %{name}-icu = %{version}-%{release}
+
 %description	devel
 It contains the libraries and header files to create applications
 
 %prep
-%setup -q
+%autosetup
+
 %build
 %configure
-make %{?_smp_mflags}
+%make_build
 
 %install
-make DESTDIR=%{buildroot} install
-find %{buildroot} -name '*.la' -delete
+%make_install
+find %{buildroot} -type f -name "*.la" -delete -print
 
 %check
-make %{?_smp_mflags} -k check
-
-%post
-/sbin/ldconfig
```

Comment:
This and the `%postun` section should not be removed. Now we're running `/sbin/ldconfig` inside the `%check` section instead of the `%post*` ones.

Reply:
I see, then same bug in dev branch too. https://github.com/microsoft/CBL-Mariner/blob/dev/SPECS/harfbuzz/harfbuzz.spec#L48